config/settings.yml

```diff
@@ -1138,4 +1139,3 @@ bid:
 # Settings for Test User Dashboard modules
 test_user_dashboard:
   env: dev
-  
```

Comment:
I have no idea why this is happening. I have 1142 lines in my local `settings.yml`, with line 1142 being a newline.